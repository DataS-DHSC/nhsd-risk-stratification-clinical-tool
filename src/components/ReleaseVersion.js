import React from 'react';
import { useFetch } from '../utils/hooks';

const ReleaseVersion = () => {
  const [siteData, loading] = useFetch({ url: '/metadata/site-data.json' });
  return (
    <>
      {`Version ${
        loading
          ? '==REPLACE_RELEASE_VERSION_TOKEN=='
          : siteData['release-version']
      }`}
    </>
  );
};

export default ReleaseVersion;
