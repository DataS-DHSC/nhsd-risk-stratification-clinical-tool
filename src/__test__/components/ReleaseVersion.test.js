import React from 'react';
import { render } from '@testing-library/react';
import ReleaseVersion from '../../components/ReleaseVersion';

const setup = () => render(<ReleaseVersion />);

describe('ReleaseVersion', () => {
  it('should match snapshot ', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
