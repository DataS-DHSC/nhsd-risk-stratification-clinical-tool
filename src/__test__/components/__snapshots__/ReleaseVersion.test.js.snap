// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReleaseVersion should match snapshot  1`] = `
<DocumentFragment>
  Version ==REPLACE_RELEASE_VERSION_TOKEN==
</DocumentFragment>
`;
